app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -114,6 +114,9 @@ data class OverdueAppointment(
     val appointmentFacilityName: String?
 ) : Parcelable {
 
+  val isAppointmentAtAssignedFacility: Boolean
+    get() = patientAssignedFacilityUuid == appointment.facilityUuid
```

Comment:
The `patientAssignedFacilityUuid` will be `null` in a lot of cases. Will this cause any issues if we don't have a null check?

Reply:
Nope, it should be `false` in those cases.